rbac/rbac/urls.py

```diff
@@ -32,6 +34,8 @@
     if not API_PATH_PREFIX.endswith("/"):
         API_PATH_PREFIX = API_PATH_PREFIX + "/"
 
+WSS_PATH_PREFIX = re.sub("api", "wss", API_PATH_PREFIX)
```

Comment:
Oh, so this is to support FE to keep the request /api/v1/cross-account-request, but get a websocket connection?

Reply:
Not quite. This just creates a new `WSS_PATH_PREFIX` using the `API_PATH_PREFIX` in the event we update/version the API. This will just swap `/api/rbac/v1/` to `/wss/rbac/v1`, which is supported in the gateway.

Reply:
Got it 👍